src/api/matrix.js

```diff
@@ -0,0 +1,33 @@
+import Http from '../http';
+import Navigator from '../navigator';
+
+class Matrix {
+  static async getMatrixDistances(routingOptions={}) {
+    if (!routingOptions.origins) {
+      const { latitude, longitude } = await Navigator.getCurrentPosition();
+      routingOptions.origins = [{ latitude, longitude }];
+    }
+
+    let {
+      origins,
+      destinations,
+      mode,
+      units,
+    } = routingOptions;
+
+    origins = origins.map(origin => { return `${origin.latitude}, ${origin.longitude}` }).join('|')
+    destinations = destinations || []
+    destinations.map(destination => { return `${destination.latitude}, ${destination.longitude}` }).join('|')
```

Comment:
not sure why our linter didn't catch this but we should have `;` on the end of these statements